shard-operator/src/main/kubernetes/minikube.yml

```diff
@@ -22,6 +22,15 @@ rules:
     verbs:
       - "get"
       - "list"
+  - apiGroups:
+      - ""
+      - "extensions"
```

Comment:
Why do we need this group for `services` and `deployments`? I think we can remove it. Also, I left a comment below in this file to uncomment and add the required resources there. I forgot to tell you in my previous comments. 😅